docs/pages/versions/unversioned/sdk/audio.mdx

```diff
@@ -31,12 +31,7 @@ You can configure `expo-audio` using its built-in [config plugin](/config-plugin
 {
   "expo": {
     "plugins": [
-      [
-        "expo-audio",
-        {
-          "microphonePermission": "Allow $(PRODUCT_NAME) to access your microphone."
-        }
-      ]
+      ["expo-audio", { "microphonePermission": "Allow $(PRODUCT_NAME) to access your microphone." }]
```

Comment:
I am not sure if this change is intentional. Can you please check?

Reply:
yeah was unintended, good catch - thanks!